frontend/src/APIClients/LogRecordAPIClient.ts

```diff
@@ -0,0 +1,39 @@
+import AUTHENTICATED_USER_KEY from "../constants/AuthConstants";
+import { getLocalStorageObjProperty } from "../utils/LocalStorageUtils";
+import baseAPIClient from "./BaseAPIClient";
+
+const createLog = async(
+    userId: number,
+    residentId: number,
+    flagged: boolean,
+    note: string,
+    attentionTo: number,
+    building: string,
+) : Promise<any> => {
```

Comment:
Would be nice to define a type for what this returns, you can do it like this with Axios:
`await baseAPIClient.post<YourTypeHere>()`

Should probably do this with the rest of the API clients as well, will look at making an issue